app/mutations/application_review_step/createReviewCommentMutation.ts

```diff
@@ -1,17 +1,28 @@
 import {graphql, DeclarativeMutationConfig} from 'react-relay';
 import RelayModernEnvironment from 'relay-runtime/lib/store/RelayModernEnvironment';
+import {ConnectionHandler} from 'relay-runtime';
 import {
   createReviewCommentMutationVariables,
   createReviewCommentMutation as createReviewCommentMutationType
 } from 'createReviewCommentMutation.graphql';
 import BaseMutation from 'mutations/BaseMutation';
 
 const mutation = graphql`
-  mutation createReviewCommentMutation($input: CreateReviewCommentInput!) {
+  mutation createReviewCommentMutation(
+    $input: CreateReviewCommentInput!
+    $connections: [ID!]!
+  ) {
     createReviewComment(input: $input) {
-      reviewCommentEdge {
+      reviewCommentEdge @appendEdge(connections: $connections) {
```

Comment:
@dleard FYI this is a handy [declarative alternative](https://relay.dev/docs/guided-tour/list-data/updating-connections/#using-declarative-directives) to the imperative updater function you were previously helping me with for removing/adding to connections (works with `@deleteEdge` as well).